lib/cc/engine/markdownlint.rb

```diff
@@ -49,8 +53,10 @@ def mdl_options
         options
       end
 
+      EXPECTED_PATTERN = /(?<path>.*):(?<line_number>\d+): (?<code>MD\d+) (?<description>.*)/
```

Comment:
Bikeshedding this name: "expected" doesn't indicate what this text describes. `ISSUE_PATTERN`, or something like that, maybe?